x-pack/filebeat/input/azureeventhub/input.go

```diff
@@ -167,6 +167,12 @@ func (a *azureInput) processEvents(event *eventhub.Event, partitionID string) bo
 func (a *azureInput) parseMultipleMessages(bMessage []byte) []string {
 	var mapObject map[string][]interface{}
 	var messages []string
+	// clean up the message for known issues producing a malformed JSON
+	if a.config.SanitizeOptions != nil {
+		for _, opt := range a.config.SanitizeOptions {
+			bMessage = sanitize(string(bMessage), opt)
```

Comment:
What if an unknown/invalid opt is passed here?

Reply:
My idea is to handle this in the sanitize function. If an unkwown option is passed, it should just return the JSON string as it was.